packages/hash/datastore/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@hashintel/hash-datastore",
+  "version": "0.0.0",
+  "private": true,
+  "description": "The data store for HASH",
+  "repository": {
+    "url": "https://github.com/hashintel/dev/packages/hash/datastore",
+    "directory": "packages/hash/datastore"
+  },
+  "license": "GPL-3.0",
+  "scripts": {
+    "fix:eslint": "eslint --ext .ts --fix .",
+    "lint:eslint": "eslint --ext .ts .",
+    "lint:tsc": "tsc --noEmit",
+    "pg:generate-ids": "ts-node postgres/scripts/generate-ids.ts",
+    "pg:migrate": "ts-node node_modules/.bin/node-pg-migrate -d HASH_MIGRATION_PG_URL --envPath ../../../.env -f ./postgres/migration-config.json",
```

Comment:
This was the quickest way for me to get this up and running. I've had a brief discussion w/ @kachkaev that we would ideally define the scripts that run our migrations in a collection of `scripts/*.ts` files, so that we can better control the importing/passing of environment variables (by importing `packages/hash/backend-utils/src/environment.ts` for instance). This would likely involve calling `node-pg-migrate` using `execa`.

To reduce the scope of this PR, and get it done quicker I've managed to get migration scripts working without these wrapper scripts, by interfacing with `node-pg-migrate` directly in the `datastore/package.json` using the `pg:migrate` shorthand. I've created [a follow-up task](https://app.asana.com/0/0/1202032662538879/f) to capture this, but I'm not sure about the priority. This should probably be done when we run into some kind of limitation with the existing migration configuration, where a custom script would solve our issue.

Reply:
I think we'd be able to call down to `node-pg-migrate`'s [programmatic API](https://salsita.github.io/node-pg-migrate/#/api) which would mean we don't need to CLIs and env vars. I think that would be worthwhile to do in that follow-up task.

Reply:
Yep, and once we've migrated to a custom script, we can also git rid of `HASH_MIGRATION_PG_URL`.

Reply:
I captured this in https://app.asana.com/0/1200211978612931/1202032662538879/f (internal)